### ISSUE TITLE
[Feature Request]: Multiple Comparisons correction for Correlation Module

### ISSUE THREAD

**Issue by iandol** [jasp-stats/jasp-issues#3062]
### Description

If I have 4 variables then 12 significance tests are performed. There should be an option to perform a multiple comparisons correction.

### Use-case

Correct correlation tables for multiple comparison

### Is your feature request related to a JASP module?

Regression

### Describe the solution you would like

As for ANOVA etc. have some options for several corrections.

### Describe alternatives that you have considered

I just do Bonferroni correction manually, but I'm sure there are better options

**Comment by TarandeepKang** [comment]
Good idea! For a function that might enable this with regard to correlations see: https://cran.r-project.org/web/packages/TestCor/index.html

**Comment by vinschger** [comment]
great idea! would be highly welcome!
